app/components/Thread/index.jsx

```diff
@@ -11,6 +11,8 @@ const mui = require('material-ui');
 const md = require('../SmartEditor/markdown');
 const LoginStore = require('../../stores/LoginStore');
 
+require('./style.less');
```

Comment:
I was tring to modify the scroll behaviour of the comments list, but gave up finally and forgot to remove this line of code. [When more comments exists, the input element will be pushed down outside the screen].